submissions/tatiana-jess/scripts/article.js

```diff
@@ -0,0 +1,60 @@
+var articles = [];
+
+function Article (opts) {
+  // : Use the js object passed in to complete this contructor function:
+  // Save ALL the properties of `opts` into `this`.
+  this.author = opts.author;
+  this.title = opts.title;
+  this.category = opts.category;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
+}
+
+Article.prototype.toHtml = function() {
+  var $newArticle = $('article.template').clone();
+  $newArticle.attr('data-category', this.category);
+
+  $newArticle.find('h1').html(this.title);
+  $newArticle.find('a[href]').html(this.authorUrl);
+  $newArticle.find('a').html(this.author);
+  $newArticle.find('time[pubdate]').html(this.publishedOn);
```

Comment:
Same here on line 21. The selector is indeed correct. So then we can call attr('datetime', this.publishedOn)